telegraf/rootfs/Dockerfile

```diff
@@ -1,20 +1,22 @@
-FROM alpine:3.2
+FROM alpine:3.3
 MAINTAINER Jonathan Chauncey "<jchauncey@deis.com>"
 
-ENV TELEGRAF_VERSION "0.10.0_linux_amd64"
+ENV TELEGRAF_VERSION=0.11.1
+ENV DOCKERIZE_VERSION=0.2.0
+RUN \
+    mkdir -p /usr/local/bin/ &&\
+    apk --update add curl &&\
+    curl -SL https://github.com/jwilder/dockerize/releases/download/v${DOCKERIZE_VERSION}/dockerize-linux-amd64-v${DOCKERIZE_VERSION}.tar.gz \
```

Comment:
string splitting as I did not realize envtpl had sprig due to it's non-existent documentation